docs/user/async-job-calls.md

```diff
@@ -114,6 +116,42 @@ Content-Type: application/json
 42
 ```
 
+### 3. Checking status of the task
```

Comment:
Just for clarification, this is provided for free by Racetrack, you don't need to update the jobtype to include this?

Reply:
Yes, it's in the Racetrack's "core".